modules/core/src/lib/deck.ts

```diff
@@ -613,6 +614,20 @@ export default class Deck {
     return this._pick('pickObjects', 'pickObjects Time', opts);
   }
 
+  addWidget(
```

Comment:
Fine but just a control question, do we need these wrappers? Can applications just use the widget manager? Especially if the next step is to add a `deck.setProp({widgets})` interface then these will rarely be called?

Also what does it mean if the widget manager is undefined - we just silently ignore these calls? It is rare enough that we don't need to worry about it?

Reply:
Removed for now, will revisit in the PR that adds declarative support